src/Tizen.NUI/src/public/BaseComponents/DirectRenderingGLView.cs

```diff
@@ -241,20 +241,23 @@ public void BindTextureResources(List<Texture> textures)
                 if (textures != null)
                 {
                     int count = textures.Count;
-                    int intptrBytes = checked(Marshal.SizeOf(typeof(IntPtr)) * count);
-                    if (intptrBytes > 0)
+                    if (count > 0)
                     {
                         IntPtr[] texturesArray = new IntPtr[count];
                         for (int i = 0; i < count; i++)
                         {
                             texturesArray[i] = HandleRef.ToIntPtr(Texture.getCPtr(textures[i]));
                         }
-                        IntPtr unmanagedPointer = Marshal.AllocHGlobal(intptrBytes);
+                        IntPtr unmanagedPointer = Marshal.AllocHGlobal(Marshal.SizeOf(typeof(IntPtr)) * texturesArray.Length);
```

Comment:
It will make another SVACE issue, what we already resolved before, by `checked` keyword.

See https://github.com/Samsung/TizenFX/pull/5465/commits/fcc64fb899494660308c0e62b3a443b2659e2c16#diff-1dc34c645af6281c23a6d3cd2f65e842458911e1f1d04df3c2a81c5e85bb8299

Please don't make svace issue occured again